mozilla/fxa
# feat(cad): Add support for auth generated CAD QR image

Fixes #5223
Fixes #5237

This PR adds support for the auth-server to generate a sign-in code qr image as well as the content-server to display the new image.

* If user is not signed into browser a default qr image is shown that doesn't prefill login forms
* No updates to content-server functional tests because to fully test this, a mobile device has to scan image
* QR code image embeds the same link that is sent via SMS

### Testing

1. Open page http://localhost:3030/post_verify/cad_qr/scan_code
2. Point your mobile device camera to qr code (it should prompt to open link)
3. Open link, redirected to App Store to download Firefox.

Unfortunately, you won't be able to test the form prefilling because the code generated was only for local dev.
